src/snowflake/connector/auth_keypair.py

```diff
@@ -45,6 +44,10 @@ def __init__(self, private_key):
         self._private_key = private_key
         self._jwt_token = ""
         self._jwt_token_exp = 0
+        self._lifetime = timedelta(seconds=120)
+
+    def change_token_lifetime(self, lifetime_in_seconds):
```

Comment:
Please use type hints for every function that you touch. [More info](https://docs.python.org/3/library/typing.html)
For this function it should be like:
```suggestion
    def change_token_lifetime(self, lifetime_in_seconds: int) -> None:
```

Reply:
Removed this function entirely since now it's set in the constructor for KeyPairAuth